Dockerfile.golang

```diff
@@ -26,6 +26,7 @@ RUN npm install -g serverless@1.30.0
 ENV TZ=Europe/London 
 ENV NODE_ENV=dev
 ENV AWS_DEFAULT_REGION=eu-west-2
+ENV GO111MODULE=on
```

Comment:
What would happen to all other projects that are currently not using modules? (like dmd, gp-profiles, etc)

Reply:
It's a good point. you can read more by running `go help module-get`. It looks like it changes behaviour (what we don't want to do for other repo). Can we set project level env GO111MODULE=on on Circleci?